src/components/Dropdown/Dropdown.tsx

```diff
@@ -287,7 +294,13 @@ export default class Dropdown extends AutoControlledComponent<
                         variables,
                       )
                     : this.renderTriggerButton(styles, getToggleButtonProps)}
-                  {toggleButton && this.renderToggleButton(getToggleButtonProps, classes, isOpen)}
+                  {Indicator.create(toggleIndicator, {
+                    defaultProps: {
+                      direction: isOpen ? 'top' : 'bottom',
+                      onClick: getToggleButtonProps().onClick,
```

Comment:
In the Indicator we are not handling the `onClick` prop, so we won't have the user props as a second arg. Will that be a problem in this case?

Reply:
Nope, it comes from Downshift